shell/common/node_bindings.cc

```diff
@@ -122,6 +123,58 @@ void stop_and_close_uv_loop(uv_loop_t* loop) {
 
 bool g_is_initialized = false;
 
+bool IsPackagedApp() {
+  base::FilePath exe_path;
+  base::PathService::Get(base::FILE_EXE, &exe_path);
+  base::FilePath::StringType path = base::ToLowerASCII(exe_path.value());
+
+#if defined(OS_WIN)
+  return path != FILE_PATH_LITERAL("electron.exe");
+#else
+  return path != FILE_PATH_LITERAL("electron");
```

Comment:
Wouldn't this return the path to helper executable on macOS, so this comparison will not be valid then.